src/lora.cpp

```diff
@@ -0,0 +1,61 @@
+#include "lora.hpp"
+#include <ArduinoJson.h>
+#include <Base64.h>
+#include <Arduino.h>
+#include "result.hpp"
+
+
+namespace obc{
+
+HardwareSerial Serial5(PD2, PC12);
+
+Result<Unit, Errc> init_lora()
+{
+    constexpr auto baud_rate = 9600l;
+    constexpr std::array arr2 = {"AT+DR=EU868", "AT+CH=NUM,0-2", "AT+MODE=LWOTAA","AT+JOIN"};
+    Serial5.begin(baud_rate);
+
+    for (const auto elem : arr2) {
+        Serial5.println(elem);
+        delay(10000);
```

Comment:
ten delay między kolejnymi komendami jest potrzebny?

Reply:
Wydaje mi się, że tutaj ze względu na komunikację z innym MCU ten delay jakiś musi pozostać, ale 10 s to chyba trochę za dużo. Może da się zmniejszyć?